api/services/elastigroup/aws/schemas/elastigroup-compute.yaml

```diff
@@ -213,6 +213,15 @@ properties:
           * The following instance types are supported by PV image: M1, M2, M3, C1, C3.
           * In case of conflict between Instance type to image type, an error message will be returned
         example: "ami-01e24be29428c15b2"
+      images:
+        type: array
+        description: |
+          Elastigroup can be configure with imageId or images, but not with both of them.
+          * Valid values: null, Array with a minimum of 1 element. <br>
+          * Multiple AMIs aren't allowed with stateful groups / MI
+          Or with the following third-party integrations: Ocean / Ecs / Nomad / Docker swarm / Kubernetes
```

Comment:
An array of image objects.
Elastigroup can be configured with either imageId or images, but not both.
* Valid values: null, or an array with at least 1 element. <br>
* Multiple AMIs aren't allowed with stateful groups / Managed Instance / Ocean
or with the following third-party integrations: ECS / Nomad / Docker Swarm / Kubernetes